ewkcoffea/modules/corrections.py

```diff
@@ -244,6 +244,8 @@ def run3_electrons_sf_attach(electrons,year,wp):
         sf_flat_75 = ceval["Electron-ID-SF"].evaluate(n_year,"sf","RecoAbove75",eta_flat,pt_flat_75)
         sf_flat_75_hi = ceval["Electron-ID-SF"].evaluate(n_year,"sfup","RecoAbove75",eta_flat,pt_flat_75)
         sf_flat_75_lo = ceval["Electron-ID-SF"].evaluate(n_year,"sfdown","RecoAbove75",eta_flat,pt_flat_75)
+    else:
+        raise Exception("Unidentified Run 3 year. Exciting!")
```

Comment:
Is this supposed to say "Exiting"? Though I guess encountering this exception could be somewhat exciting as well (though not necessarily in a good way :) )

Reply:
I was going for exciting 😂